azerothcore/azerothcore-wotlk
# fix(Core/Spells): Add more exceptions to Aura of Despair.

## Changes Proposed:
-  Add more exceptions to Aura of Despair (General Vezax)


## SOURCE:

https://classic.warcraftlogs.com
(Water shield should also restore mana, but only on searing flames, not on profound darkness, as it does now)

https://classic.warcraftlogs.com/reports/yq3jLX8762abghYA#fight=61&type=resources&spell=100
- Improved Felhunter
- Blessing of Sanctuary (prot pala)
- Dark Rune (fire mage)
- Demonic Rune (fire mage)
- Improved Leader of the Pack (feral)
## Tests Performed:

- tested


## How to Test the Changes:


.go c id 33271
Confirm that you gain mana with one of the added spells and items
.additem 12662 -- Demonic Rune
.additem 20520 -- Dark Rune
.aura 20911 -- Blessing of Sanctuary
Make a feral druid, talent into LoP and Imp. LoP
Make a warlock and talent into Improved Felhunter
.learn all my spells


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
